numpy/tests/test_public_api.py

```diff
@@ -154,6 +154,7 @@ def test_NPY_NO_EXPORT():
     "lib.mixins",
     "lib.recfunctions",
     "lib.scimath",
+    "lib.stride_tricks",
```

Comment:
With [`as_strided`](https://numpy.org/doc/stable/reference/generated/numpy.lib.stride_tricks.as_strided.html), and the newly introduced [`sliding_window_view`](https://numpy.org/doc/stable/reference/generated/numpy.lib.stride_tricks.sliding_window_view.html) function, there are currently 2 
public objects in `lib.stride_tricks` that can:
a. Only be imported from a private module.
b. Are publicly documented to-be imported from aforementioned module.

Because of this `lib.stride_tricks` has now been (formally) moved to the list of public modules.

Reply:
@rgommers Thoughts about adding `lib.stride_tricks` to the list of public modules?

Reply:
Makes sense to me.